### ISSUE TITLE
macOS Build process fails

### ISSUE THREAD

**Issue by camdenmoors** [oclif/oclif#865]
Do you want to request a *feature* or report a *bug*?
-----------------------------------------------------
Report a bug

What is the current behavior?
-----------------------------
Trying to build a macOS package (x86 and M1) on Github  actions (and locally) throws a `ReferenceError: arch is not defined` after updating to oclif v3.0.0.

What is the expected behavior?
------------------------------
Both macOS packages should build correctly.

---

Our build action: https://github.com/mitre/saf/blob/main/.github/workflows/build-macos.yml

Relevant log:
```
added 450 packages, and audited 451 packages in 16s

30 packages are looking for funding
  run `npm fund` for details

8 high severity vulnerabilities

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
oclif: building target saf-v1.1.4-be0467a-darwin-x64.tar.gz
oclif: copying workspace ./tmp/saf ./tmp/darwin-x64/saf
oclif: downloading node-v16.14.2-darwin-x64
node-v16.14.2-darwin-x64.tar.xz: OK
oclif: extracting node-v16.14.2-darwin-x64
oclif: building target saf-v1.1.4-be0467a-darwin-arm64.tar.gz
oclif: copying workspace ./tmp/saf ./tmp/darwin-arm64/saf
oclif: downloading node-v16.14.2-darwin-arm64
node-v16.14.2-darwin-arm64.tar.xz: OK
oclif: extracting node-v16.14.2-darwin-arm64
    ReferenceError: arch is not defined
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by RodEsp** [comment]
Heya @camdenmoors would you happen to have an `oclif.update.node.targets` property defined in your `package.json`?

**Comment by RodEsp** [comment]
This is now fixed in `v3.0.1`!
